.github/workflows/ci-build.yml

```diff
@@ -37,36 +37,28 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: "1.22"
+          go-version: "1.23"
           cache-dependency-path: "go-client/go.sum"
-      - name: Cache SonarCloud packages
```

Comment:
Can you explain, why caching has been removed?

Reply:
We use a GH Action for Sonar `SonarSource/sonarqube-scan-action` and I assume that it takes care of caching internally ;)